new/components/restservice/config/cloudify-rest.conf

```diff
@@ -6,10 +6,13 @@
     rest_service_log_files_backup_count: 20,
     db_address: '{{ ctx.node.properties.es_endpoint_ip }}',
     db_port: '{{ ctx.node.properties.es_endpoint_port }}',
-    amqp_address: 'localhost',
+    amqp_address: 'localhost:(( amqp_port ))/',
```

Comment:
again, I think we should use runtime properties and download-resource-and-render for these instead of injecting them.